source/docs/software/advanced-gradlerio/compiler-args.rst

```diff
@@ -0,0 +1,39 @@
+Using Compiler Arguments
+============================
+
+Compiler arguments allow us to change the behavior of our compiler. This includes making warnings into errors, ignoring certain warnings and choosing optimization level. Normally we would pass them in as flags when compiling our code but this doesn't work in GradleRIO. Instead we have to modify our build.gradle.
+
+C++
+---
+
+Platforms
+^^^^^^^^^
+
+In order to avoid breaking different platforms with compiler flags we configure everything per platform. The platforms that are supported are :
```

Comment:
I would lead in by saying different compilers on different platforms use different flags, and then say to avoid breaking